src/metabase/api/search.clj

```diff
@@ -598,6 +639,7 @@
    created_by                          [:maybe (ms/QueryVectorOf ms/PositiveInt)]
    last_edited_at                      [:maybe ms/NonBlankString]
    last_edited_by                      [:maybe (ms/QueryVectorOf ms/PositiveInt)]
+   model-parents                       [:maybe :boolean]
```

Comment:
just noticing that we have underscores for some variables but a hyphen in this one

Reply:
Excellent point, thank you for noticing this!

Reply:
A literally nit-sized suggestion!

Reply:
Would it be easy to add the ids as well as the names of the ancestor collections?

Speaking of which I wonder if something like "include_collection_ancestors" would be worth considering

Reply:
Yep, that would be easy enough! What shape would we want here? Maybe something like:
```
{:ancestor_collections [{:id 1 :name "foo"} {:id 2 :name "bar"}]}
```
instead of
```
{:collection_names ["foo" "bar"]}
```
?

Reply:
That looks good! Hm, maybe the final collection shouldn't be included in this list since it is already available on `collection`, and it's not strictly speaking an 'ancestor'? Or is the whole list considered the 'ancestry' of the model?

Another variable name that might work is 'collection_path'

Reply:
I think I'd say that a parent collection is an ancestor, but I'm fine just including grandparents and above if it makes your life easier!